src/Toolkit.Forms/PopupViewer/PopupViewerRenderer.cs

```diff
@@ -28,7 +28,7 @@
 
 namespace Esri.ArcGISRuntime.Toolkit.Xamarin.Forms
 {
-    internal class PopupViewerRenderer : ViewRenderer<PopupViewer, UI.Controls.PopupViewer>
+    internal class PopupViewerRenderer : ViewRenderer<PopupViewer, UI.Controls.PopupViewer?>
```

Comment:
Why is `TNativeView` typeparam marked nullable here, and not in other renderers (e.g. LayerLegendRenderer or CompassRenderer)?